### ISSUE TITLE
Sentinel-3 image search error

### ISSUE THREAD

**Issue by skualos** [semiautomaticgit/SemiAutomaticClassificationPlugin#104]
The following error occurs when attempting to search for Sentinel-3 images:
```
An error has occurred while executing Python code: 

AttributeError: 'str' object has no attribute 'read' 
Traceback (most recent call last):
  File "/home/starch/.local/share/QGIS/QGIS3/profiles/default/python/plugins/SemiAutomaticClassificationPlugin/maininterface/downloadproducts.py", line 236, in findImages
    self.downloadMetadata()
  File "/home/starch/.local/share/QGIS/QGIS3/profiles/default/python/plugins/SemiAutomaticClassificationPlugin/maininterface/downloadproducts.py", line 244, in downloadMetadata
    self.queryDatabaseSentinel3()
  File "/home/starch/.local/share/QGIS/QGIS3/profiles/default/python/plugins/SemiAutomaticClassificationPlugin/maininterface/downloadproducts.py", line 801, in queryDatabaseSentinel3
    xml = response.read()
AttributeError: 'str' object has no attribute 'read'

Python version: 3.8.2 (default, Apr  8 2020, 14:31:25) [GCC 9.3.0] 
QGIS version: 3.12.0-București București, exported 
```

**Comment by semiautomaticgit** [comment]
Hello, please copy the plugin log file following these steps https://semiautomaticclassificationmanual.readthedocs.io/en/latest/faq.html#how-can-i-report-an-error

**Comment by skualos** [comment]
Hi. Below is the log. I think there is an issue with the URL that is being generated by SCP

Date	Function	Message
2020-04-21 13.55.38.178070	SemiAutomaticClassificationPlugin.maininterface.settings-logCheckbox 148	LOG ACTIVE SemiAutomaticClass Version 6.4.3 - Greenbelt - QGIS v. 31200 L:en - OS Linux - 64bit =Yes
2020-04-21 13.55.46.218747	SemiAutomaticClassificationPlugin.maininterface.settings-testGDAL 270	 gdal version:3.0
2020-04-21 13.55.46.220081	SemiAutomaticClassificationPlugin.maininterface.settings-testGDAL 278	 test: Success
2020-04-21 13.55.46.347534	SemiAutomaticClassificationPlugin.core.utils-GDALCopyRaster 5813	 image: /tmp/semiautomaticclassification/20200421_131511130361/20200421_135546220608test.tif
2020-04-21 13.55.46.348001	SemiAutomaticClassificationPlugin.maininterface.settings-testGDALSubprocess 313	 test: Success
2020-04-21 13.55.46.348246	SemiAutomaticClassificationPlugin.maininterface.settings-testNumpy 344	 test: Success
2020-04-21 13.55.46.348459	SemiAutomaticClassificationPlugin.maininterface.settings-testScipy 355	 test: Success
2020-04-21 13.55.46.348670	SemiAutomaticClassificationPlugin.maininterface.settings-testMatplotlib 366	 test: Success
2020-04-21 13.55.46.348926	SemiAutomaticClassificationPlugin.core.utils-downloadFile 347	 url:https://semiautomaticgit.github.io/SemiAutomaticClassificationPluginWelcome/changelog.html
2020-04-21 13.55.46.349485	SemiAutomaticClassificationPlugin.core.utils-getProxyHandler 313
2020-04-21 13.55.46.854925	SemiAutomaticClassificationPlugin.maininterface.settings-testInternetConnection 325	 test: Success
2020-04-21 13.56.29.092007	SemiAutomaticClassificationPlugin.core.input-menuIndex 435
2020-04-21 13.56.29.150406	SemiAutomaticClassificationPlugin.core.input-menuClicked 446
2020-04-21 13.56.31.608727	SemiAutomaticClassificationPlugin.core.input-menuIndex 435
2020-04-21 13.56.31.694731	SemiAutomaticClassificationPlugin.core.input-menuClicked 446
2020-04-21 13.56.36.610301	SemiAutomaticClassificationPlugin.maininterface.downloadproducts-pointerActive 113	pointer active: Landsat
2020-04-21 13.56.45.792107	SemiAutomaticClassificationPlugin.core.utils-passwordConnectPython 246	 https://scihub.copernicus.eu/apihub/search?q=*efr*%20AND%20beginPosition:[2016-01-01T00:00:00.000Z%20TO%202020-04-21T23:59:59.999Z]%20AND%20footprint:"Intersects(POLYGON((115.92877692433%20-27.15588891638,115.92877692433%20-29.86153939606,119.27758696870%20-29.86153939606,119.27758696870%20-27.15588891638,115.92877692433%20-27.15588891638)))%22&rows=20
2020-04-21 13.56.45.792858	SemiAutomaticClassificationPlugin.core.utils-getProxyHandler 313
2020-04-21 13.56.45.793523	SemiAutomaticClassificationPlugin.core.utils-getPasswordHandler 240
2020-04-21 13.57.49.425413	SemiAutomaticClassificationPlugin.core.utils-passwordConnectPython 292	 ERROR exception: HTTP Error 500: Internal Server Error
2020-04-21 13.58.36.513293	SemiAutomaticClassificationPlugin.core.input-menuIndex 435
2020-04-21 13.58.36.601008	SemiAutomaticClassificationPlugin.core.input-menuClicked 446
2020-04-21 13.58.50.027340	SemiAutomaticClassificationPlugin.maininterface.settings-logCheckbox 151	LOG DISABLED

**Comment by semiautomaticgit** [comment]
Hello, it seems they have changed something in the APIs.
I'll fix this in an update

**Comment by semiautomaticgit** [comment]
Fixed in version 6.4.4
